kvlds/btree_find.h

```diff
@@ -37,7 +37,7 @@ int btree_find_leaf(struct btree *, struct node *, const struct kvldskey *,
  * btree_find_range(T, N, k, h, callback, cookie):
  * Search for a node of height ${h} or less in the subtree of ${T} rooted at
  * ${N} which is responsible for a range including the key ${k}.  Invoke
- * ${callback}(${cookie}, L, e} with the node ${L} locked, where ${L} is the
+ * ${callback}(${cookie}, L, e) with the node ${L} locked, where ${L} is the
```

Comment:
You might have wanted to have the entire "callback example" inside `${ ... }`, but the new syntax matches the one you used for `btree_find_leaf()` 10 lines earlier.